parquet/write.go

```diff
@@ -15,17 +15,21 @@ import (
 
 func (*Client) WriteTableBatch(w io.Writer, table *schema.Table, records []arrow.Record) error {
 	props := parquet.NewWriterProperties(
+		parquet.WithVersion(parquet.V2_4),
```

Comment:
shouldn't need this the default version is V2_LATEST

Reply:
I was trying different options to see if anything comes up.